README.md

```diff
@@ -2,22 +2,98 @@
 
 [![Test action](https://github.com/LemLib/pros-build/actions/workflows/test.yml/badge.svg)](https://github.com/LemLib/pros-build/actions/workflows/test.yml)
 
-## Usage
+This action creates an environment capable of building PROS projects and templates, and builds them using [build.sh](/build-tools/build.sh)
 
-Here are the basics `steps` you'd need to build a pros project:
+Instructions on creating a custom build script, adding additional packages, and using this image as a base are located at the end of this readme.
+
+## Usage:
+
+### Inputs
+
+- `multithreading`
+  - Wether to use multithreading when building the project
+  - Default: `true`
+  - Required: `false`
+- `no_commit_hash`
+  - Wether to include a shortened commit hash at the end of the artifact name
+  - Example: `LemLib@0.5.1+5881ac`
+  - Default: `true`
+  - Required: `false`
+- `copy_readme_and_license_to_include`
+  - Whether to make a VERSION file, copy the README(.md), and copy the LICENSE(.md) files to the `/include/(library name)` folder.
+  - required: `false`
+  - default: `false`
+- `lib_folder_name`
+  - The name of the library's folder name under the include directory.
+  - required: `if copy_readme_and_license_to_include is set`
+- `write_job_summary`
+  - Whether to output build information to GitHub's Job Summary
```

Comment:
I feel like this could be misleading

Reply:
Which part

Reply:
that it just says build info, it really enables outputting all of the Job Summary